<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Search">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>34</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>94</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QListView" name="resultsListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>431</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QListView" name="infoListView">
      <property name="geometry">
       <rect>
        <x>415</x>
        <y>50</y>
        <width>351</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="likeBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>440</y>
        <width>94</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="watchLaterBtn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>94</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>WatchLater</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Liked">
     <attribute name="title">
      <string>Liked</string>
     </attribute>
     <widget class="QListView" name="likedListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WatchLater">
     <attribute name="title">
      <string>WatchLater</string>
     </attribute>
     <widget class="QListView" name="watchLaterListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
